modular_skyrat/modules/sec_haul/code/security_medic/secmed_clothes.dm

```diff
@@ -30,28 +38,16 @@
 
 /obj/item/clothing/under/rank/security/peacekeeper/security_medic
 	name = "security medics's turtleneck"
-	desc = "A comfy red turtleneck with a white armband, denoting the wearer as a security medic."
-	icon = 'modular_skyrat/master_files/icons/obj/clothing/uniforms.dmi'
-	worn_icon = 'modular_skyrat/master_files/icons/mob/clothing/uniform.dmi'
-	worn_icon_digi = 'modular_skyrat/master_files/icons/mob/clothing/uniform_digi.dmi'
+	desc = "A comfy turtleneck with a white armband, denoting the wearer as a security medic."
+	icon = 'modular_skyrat/master_files/icons/obj/clothing/under/security.dmi'
+	worn_icon = 'modular_skyrat/master_files/icons/mob/clothing/under/security.dmi'
 	icon_state = "security_medic_turtleneck"
 
 /obj/item/clothing/under/rank/security/peacekeeper/security_medic/skirt
 	name = "security medics's skirtleneck"
-	desc = "A comfy red turtleneck with a white armband and brown skirt, denoting the wearer as a security medic."
-	icon = 'modular_skyrat/master_files/icons/obj/clothing/uniforms.dmi'
-	worn_icon = 'modular_skyrat/master_files/icons/mob/clothing/uniform.dmi'
-	worn_icon_digi = 'modular_skyrat/master_files/icons/mob/clothing/uniform_digi.dmi'
+	desc = "A comfy turtleneck with a white armband and brown skirt, denoting the wearer as a security medic."
 	icon_state = "security_medic_turtleneck_skirt"
 
```

Comment:
```suggestion

/obj/item/clothing/under/rank/security/peacekeeper/security_medic/old
	name = "security medics's uniform"
	desc = "A lightly armored uniform worn by Nanotrasen's Asset Protection Medical Corps."
	icon = 'modular_skyrat/master_files/icons/obj/clothing/uniforms.dmi'
	worn_icon = 'modular_skyrat/master_files/icons/mob/clothing/uniform.dmi'
	worn_icon_digi = 'modular_skyrat/master_files/icons/mob/clothing/uniform_digi.dmi'
	icon_state = "security_medic_jumpsuit"

```
Security medic is getting this all cleaned out separately because its even more of a mess